app/main.py

```diff
@@ -1,3 +1,60 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, "Distance"]) -> "Distance":
+        if type(other) == int or type(other) == float:
```

Comment:
Use the opposite condition: `if isinstance(other, Distance)`
It's better to use `isinstance` than `type` for type checks